bridge-cli/src/omni_connector_command.rs

```diff
@@ -20,170 +20,209 @@ use crate::{combined_config, CliConfig, Network};
 
 #[derive(Subcommand, Debug)]
 pub enum OmniConnectorSubCommand {
+    #[clap(about = "Log metadata for a token")]
     LogMetadata {
-        #[clap(long)]
+        #[clap(short, long, help = "Token address to logging metadata")]
         token: OmniAddress,
         #[command(flatten)]
         config_cli: CliConfig,
     },
+
+    #[clap(about = "Deploy a token on NEAR")]
     NearDeployToken {
-        #[clap(short, long)]
+        #[clap(short, long, help = "Chain to deploy the token on")]
         chain: ChainKind,
-        #[clap(short, long)]
+        #[clap(
+            short,
+            long,
+            help = "Transaction hash of the LogMetadata call on other chain"
+        )]
         tx_hash: String,
         #[command(flatten)]
         config_cli: CliConfig,
     },
+    #[clap(about = "Deploy a token on NEAR with EVM proof")]
     NearDeployTokenWithEvmProof {
-        #[clap(short, long)]
+        #[clap(short, long, help = "Chain to deploy the token on")]
         chain: ChainKind,
-        #[clap(short, long)]
+        #[clap(
+            short,
+            long,
+            help = "Transaction hash of the LogMetadata call on other chain"
+        )]
         tx_hash: String,
         #[command(flatten)]
         config_cli: CliConfig,
     },
+    #[clap(about = "Deposit storage for a token on NEAR")]
     NearStorageDeposit {
-        #[clap(short, long)]
+        #[clap(short, long, help = "Token to deposit storage for")]
         token: String,
-        #[clap(short, long)]
+        #[clap(short, long, help = "Amount to deposit")]
         amount: u128,
         #[command(flatten)]
         config_cli: CliConfig,
     },
+    #[clap(about = "Sign a transfer on NEAR")]
     NearSignTransfer {
-        #[clap(short, long)]
+        #[clap(short, long, help = "Origin chain ID of transfer to sign")]
         origin_chain_id: u8,
-        #[clap(short, long)]
+        #[clap(short, long, help = "Origin nonce of transfer to sign")]
         origin_nonce: u64,
-        #[clap(short, long)]
-        fee_recipient: Option<String>,
-        #[clap(short, long)]
+        #[clap(short, long, help = "Fee recipient account ID")]
+        fee_recipient: Option<AccountId>,
+        #[clap(short, long, help = "Fee to charge for the transfer")]
         fee: u128,
-        #[clap(long)]
+        #[clap(short, long, help = "Native fee to charge for the transfer")]
         native_fee: u128,
         #[command(flatten)]
         config_cli: CliConfig,
     },
+    #[clap(about = "Initialize a transfer on NEAR")]
     NearInitTransfer {
-        #[clap(short, long)]
+        #[clap(short, long, help = "Token to transfer")]
         token: String,
-        #[clap(short, long)]
+        #[clap(short, long, help = "Amount to transfer")]
         amount: u128,
-        #[clap(short, long)]
+        #[clap(short, long, help = "Recipient account ID on the other chain")]
```

Comment:
"Recipient address on the destination chain" perhaps better?
Also I would rename the argument to `recipient`